.github/workflows/CreateContainer_release.yml

```diff
@@ -9,6 +9,12 @@ jobs:
     with:
       workdir: Frends.AzureBlobStorage.CreateContainer
       env_var_name_1: HiQ_AzureBlobStorage_ConnString
+      env_var_name_2: HiQ_AzureBlobStorage_AppID
+      env_var_name_3: HiQ_AzureBlobStorage_TenantID
+      env_var_name_4: HiQ_AzureBlobStorage_ClientSecret
```

Comment:
Release doesn't run tests, so all of these `env_var_name` and `env_var_value` can be removed.